doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -313,6 +313,7 @@ Bluetooth libraries and services
   * Updated:
 
     * The :kconfig:option:`BT_MESH_MODEL_SRV_STORE_TIMEOUT` Kconfig option, that is controlling timeout for storing of model states, is replaced by the :kconfig:option:`BT_MESH_STORE_TIMEOUT` Kconfig option.
+    * Fixed an issue where :ref:'bt_mesh_dtt_srv_readme' could not be found for models spanning multiple elements.
```

Comment:
```suggestion
* Fixed an issue where the :ref:'bt_mesh_dtt_srv_readme' model could not be found for models spanning multiple elements.
```

Reply:
You probably missed that I changed the indentation here :)
Move this line outside the Updated list (on the same level as Updated list).
Usually, the order of changes would be:
Added
Updated
Fixed
Removed

Reply:
I do not think that has been done before, I think they have been under updeted, but I will update it.

Reply:
Fixed